chapters/lit/main.tex

```diff
@@ -1235,9 +1237,19 @@ \section{Chapter summary}
 quality, often from an ethical standpoint where people (and their data) are the
 subject of the algorithms.
 
-Other than these practices, the evaluation of clustering (and other machine
-learning) algorithms appears to rely heavily on confirmation processes. A
-confirmation process applies a method to a specific example and quantifies its
-success using some metric. Chapter~\ref{chp:edo} introduces a novel approach to
-expand on this approach to understanding algorithm quality, and discusses the
-potential dangers of being too reliant on a small number of examples.
+Despite the widespread use of clustering in healthcare, the survey for this
+review identified no existing works where clustering was used to inform a
+queuing model. Indeed, clustering has been used to understand and categorise the
+outputs of healthcare queuing models, but does not appear to have been applied
+at the front-end of a model. In Chapter~\ref{chp:copd}, administrative data is
+utilised to profile spells associated with a healthcare population via
+clustering. These profiles are then included as classes in a queue which models
+the population in a hypothetical hospital system.
+
+The works introduced in Chapters~\ref{chp:edo}~and~\ref{chp:kmodes} are vital in
+producing the model used in Chapter~\ref{chp:copd}. As such, they exist as a
+whole, singular piece of research, forming this thesis. Importantly, each
+chapter provides novel contributions to the literature of their respective
+theme: evaluation, clustering, and healthcare modelling. However, and perhaps
+more importantly, this thesis acts as a substantial and unified contribution to
+the literature at the intersection of these themes, which is so lacking.
```

Comment:
```suggestion
the literature at the intersection of these themes, which is lacking.
```

I think? But this is the type of thing you're usually right about.

Reply:
I prefer without actually. It's a touch emotive otherwise.